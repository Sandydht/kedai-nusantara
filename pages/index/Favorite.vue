<template>
  <div id="favoritePage">
    <p>Favorite page</p>
  </div>
</template>

<script>
export default {
  name: 'Favorite'
}
</script>
