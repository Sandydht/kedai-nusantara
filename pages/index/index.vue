<template>
  <div id="homePage">
    <p>Home page</p>
  </div>
</template>

<script>
export default {
  name: 'Home'
}
</script>
