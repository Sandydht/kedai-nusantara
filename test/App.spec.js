import { shallowMount } from '@vue/test-utils'
import index from '~/pages'

describe('App.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(index)
  })

  test('has expected html structure when side-bar component render in the DOM', async () => {
    await wrapper.setData({ showSidebar: true })
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('has expected html structure when side-bar component not render in the DOM', async () => {
    await wrapper.setData({ showSidebar: false })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
