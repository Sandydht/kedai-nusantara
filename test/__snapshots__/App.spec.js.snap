// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue has expected html structure when side-bar component not render in the DOM 1`] = `
"<div id=\\"app\\">
  <app-bar-stub></app-bar-stub>
  <!---->
  <nuxt></nuxt>
</div>"
`;

exports[`App.vue has expected html structure when side-bar component render in the DOM 1`] = `
"<div id=\\"app\\">
  <app-bar-stub></app-bar-stub>
  <div>
    <side-bar-stub></side-bar-stub>
  </div>
  <nuxt></nuxt>
</div>"
`;
